src/main/java/com/programmers/calculator/engine/calculation/CalculatePostfix.java

```diff
@@ -0,0 +1,41 @@
+package com.programmers.calculator.engine.calculation;
+
+import java.util.List;
+import java.util.Stack;
+
+public class CalculatePostfix implements Calculate {
+    private final String NUMBER = "^[0-9]*$";
+    static Stack<Double> stack = new Stack<>();
+
+
+    @Override
+    public Double calculate(List<String> postfix) {
+        for (String token : postfix) {
+            isNumber(token);
+        }
+        return stack.pop();
+    }
+
+
+    //이렇게 세세하게 나누는 방법이 좋은건지 궁금합니다.
+    void isNumber(String token) {
+        if (token.matches(NUMBER)) {
+            pushStack(token);
+        } else {
+            calculateStack(token);
+        }
+    }
+
+    void pushStack(String token) {
+        stack.push(Double.valueOf(token));
+    }
+
+    void calculateStack(String token) {
+        double number1, number2;
+        number2 = stack.pop();
+        number1 = stack.pop();
+        stack.push(Symbol.calculate(token, number1, number2));
```

Comment:
`token`으로 `Symbol` 인스턴스를 생성하고 메시지를 전달하는 것이 조금 더 좋을 것 같아요
```suggestion
	Symbol symbol = Symbol.of(token);
	symbol.calculate(number1, number2);
```

Reply:
이 방법에 관하여 인스턴스를 통한 메시지 전달의 장점을 찾아봤는데 잘 이해가 안되서 질문 남깁니다
`Symbol symbol = Symbol.of(token);` 이 방법은 Symbol에 대한 값에 대한 검증을 위한건가요??

Reply:
`of`는 `Symbol`의 인스턴스를 만들기위한 정적 팩터리 메서드입니다.
https://hudi.blog/effective-java-static-factory-method/

객체를 생성하고 그 객체에게 메시지를 전달해주면 좀 더 객체지향적이라고 볼 수 있다고 생각해서 말씀드렸어요. 제가 좀 더 자세히 답변드렸어야 했네요